truss/tests/test_truss_handle.py

```diff
@@ -575,6 +575,45 @@ def predict(self, request):
         assert len(th.get_all_docker_images()) == orig_num_truss_images + 1
 
 
+@pytest.mark.integration
```

Comment:
Do these need to be integration? Sounds like they would fail quickly with exception.